kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -862,6 +941,42 @@ public boolean isV3ApiEnabled() {
     return getBoolean(API_V3_ENABLE_CONFIG);
   }
 
+  public final boolean isRateLimitEnabled() {
+    return getBoolean(RATE_LIMIT_ENABLE_CONFIG);
+  }
+
+  public final RateLimitBackend getRateLimitBackend() {
+    return RateLimitBackend.valueOf(getString(RATE_LIMIT_BACKEND_CONFIG).toUpperCase());
+  }
+
+  public final Integer getRateLimitPermitsPerSec() {
+    return getInt(RATE_LIMIT_PERMITS_PER_SEC_CONFIG);
+  }
+
+  public final Duration getRateLimitTimeout() {
+    return Duration.ofMillis(getLong(RATE_LIMIT_TIMEOUT_MS_CONFIG));
+  }
+
+  public final int getRateLimitDefaultCost() {
+    return getInt(RATE_LIMIT_DEFAULT_COST_CONFIG);
+  }
+
+  public final ImmutableMap<String, Integer> getRateLimitCosts() {
+    String value = getString(RATE_LIMIT_COSTS_CONFIG);
+    return Arrays.stream(value.split(","))
+        .filter(entry -> !entry.isEmpty())
+        .peek(
+            entry ->
+                checkArgument(
+                    entry.contains("="),
+                    String.format(
+                        "Invalid value for config %s: %s", RATE_LIMIT_COSTS_CONFIG, value)))
```

Comment:
Potentially say what the format should be too (in error messages it's generally best to give a next action as well as the error).

Reply:
Done.